test/integration_test/cloud_disk_test.go

```diff
@@ -20,6 +20,7 @@ import (
 var (
 	disksEncAzure = []string{"type=Premium_LRS,size=100,diskEncryptionSetID=invalid"}
 	disksEncGce   = []string{"type=pd-standard,size=150,kms=invalid"}
+	disksEncOke   = []string{"type=pd-standard,size=150,kms=invalid"}
```

Comment:
```suggestion
	disksEncOke   = []string{"type=pv-10,size=150,kms=invalid"}
```
Change `type` field to above.